.travis.yml

```diff
@@ -1,13 +1,12 @@
-sudo: false
```

Comment:
The sudo keyword was deprecated and the build works fine without it. 

https://changelog.travis-ci.com/linux-builds-run-on-vms-by-default-77106